internal/backend/runtime/omni/audit/auditlog/auditlogfile/log_file.go

```diff
@@ -169,6 +169,7 @@ func (l *logReader) Close() error {
 func (l *logReader) Read() ([]byte, error) {
 	if l.scanner == nil {
 		l.scanner = bufio.NewScanner(l.rdr)
+		l.scanner.Buffer(nil, 4*1024*1024) // 4MB max line size
```

Comment:
should we go with `16 MB` to be on the safe side (or even more given that we know there might be some really long lines)? it doesn't cost us much